rust/src/combinators/measure_cast/bounded_range_to_pureDP/pseudocode/make_bounded_range_to_pureDP.py

```diff
@@ -0,0 +1,7 @@
+# type: ignore
+def make_bounded_range_to_pureDP(meas: Measurement) -> Measurement:
+    return meas.with_map( # |\label{with_map}|
```

Comment:
FWIW, the `#` (but not the following text) does get rendered in the PDF. File a very-low priority issue?

Reply:
The hash is annoying, not sure how I'd fix it, but it doesn't strike me as particularly problematic and arguably gives a little indication where a label is.